tests/group/nested_groups.robot

```diff
@@ -0,0 +1,120 @@
+*** Settings ***
+Resource  lib/utils.robot
+
+Suite Setup  Go to IAM
+Suite Teardown  Close All Browsers
+
+Test Setup  Setup for nested group tests
+Test Teardown  Teardown for nested group tests
+
+*** Variables ***
+${PARENT_GROUP}  Test-001
+${L1_GROUP}      test-L1
+${L2_GROUP}      test-L2
+
+  
+*** Test Cases ***
+Create group with parent
+  Open Groups Add Group Dialog
+  Create group with parent  ${L1_GROUP}  ${PARENT_GROUP}
+  Find group in groups page  ${PARENT_GROUP}/${L1_GROUP}
+  
+Create new subgroup
+  Create Subgroup  ${L1_GROUP}  ${PARENT_GROUP}
+  ${row}=  Set Variable  2
+  ${id}=  Set Variable  subgroupslist
+  Table Cell Should Contain  ${id}  ${row}  1  1
+  Table Cell Should Contain  ${id}  ${row}  2  ${L1_GROUP}
+  Table Cell Should Contain  ${id}  ${row}  3  Remove
+  
+Create second level subgroup
+  Open Groups Add Group Dialog
+  Create group with parent  ${L1_GROUP}  ${PARENT_GROUP}
+  Open Groups Add Group Dialog
+  Create group with parent  ${L2_GROUP}  ${PARENT_GROUP}/${L1_GROUP}
+  ${fullname}=  Set Variable  ${PARENT_GROUP}/${L1_GROUP}/${L2_GROUP}
+  Find group in groups page  ${fullname}
+  Delete subgroup  ${fullname}
+  
```

Comment:
We could add the test "Delete not empty subgroup" and try to remove the first sublevel and check the error